roxctl/collector/supportpackages/upload/command.go

```diff
@@ -24,8 +29,9 @@ type collectorSPUploadCommand struct {
 func Command(cliEnvironment environment.Environment) *cobra.Command {
 	collectorSPUploadCmd := &collectorSPUploadCommand{env: cliEnvironment}
 	c := &cobra.Command{
-		Use:   "upload <package-file>",
-		Short: "Upload files from a collector support package to Central.",
+		Use:        "upload <package-file>",
+		Short:      "Upload files from a collector support package to Central.",
```

Comment:
nit: You are capitalizing the `Collector` in the other short message and here you are not.